config/sync/migrate_plus.migration.stanford_localist_importer.yml

```diff
@@ -494,6 +494,46 @@ process:
           entity_type: taxonomy_term
           ignore_case: true
           source: name
+  image_file:
+    -
+      plugin: skip_on_empty
+      method: process
+      source: photo_url
+    -
+      plugin: file_import
+      destination: constants/image_path
+      id_only: true
+      reuse: true
```

Comment:
After doing some adjustments for H&S very recently, i found a slightly better approach to this. We've seen sometimes the image url from Localist will change without the image actually changing. So we decided to construct the destination to use the event machine name from localist.
[See here](https://github.com/SU-HSDO/suhumsci/blob/develop/config/default/migrate_plus.migration.hs_localist_individual.yml#L277). You should be able to copy the whole of that down to line 317.

Reply:
I plugged this in and re-ran the imports. Seems good. I got a "Could not retrieve the oEmbed resource." But the image is there. 🤷

Reply:
meh, that could also be from something else unrelated too.